charts/spire/charts/spire-server/values.yaml

```diff
@@ -182,15 +182,21 @@ upstreamAuthority:
         key: ""
         bundle: ""
   certManager:
-    enabled: false
+    enabled: true
     rbac:
       create: true
-    issuer_name: "spire-ca"
+    issuer_name: "spire-server-ca"
     issuer_kind: "Issuer"
     issuer_group: "cert-manager.io"
     # -- Specify to use a namespace other then the one the chart is installed into
     namespace: ""
     kube_config_file: ""
+
+    createCA: false
+    signed:
+      spec:
+        selfSigned: {}
```

Comment:
Could we include a commented block to show how to use this setting.

Reply:
If you set selfSigned: {} here, then there is no option to not do selfSigned: {}

Reply:
yes I know, but for our chart users it might not be obvious how this is used in the cert-manager CRD.

Reply:
Did a suggestion above to document this value. Please ensure to update the docs after applying.